BUILD.md

```diff
@@ -427,3 +428,9 @@ build_architecture
   }
 build_architecture
 </details>
+
+# Generating CLI docs
```

Comment:
How would we use the generated YAML from the command with the `linkerd/website` repo? I think we may need additional documentation to describe the `doc` command's If that doesn't exist already somewhere.

Reply:
It hasn't been wired up yet, but will look something like https://github.com/linkerd/website/blob/master/linkerd.io/data/cli.yaml. My preference would be to have all the documentation on how to do it in the website repo (since that's the end display case).